nullaway/build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
     testCompile deps.apt.javaxInject
     testCompile deps.test.rxjava2
 
-    epCore deps.build.errorProneCore
+    epJavac "com.google.errorprone:javac:9-dev-r4023-3"
```

Comment:
@kageiit what I really want to here is reference the version of `com.google.errorprone:javac` that `deps.build.errorProneCore` is using.  Is there any way to do that?

Reply:
Yes. There is a way to extract this version. I can fix it up in a follow up or just push to the same branch. lemme know

Reply:
If you could push on the branch, that'd be great, thanks